rpm/toggle-monitor/toggle-monitor.spec

```diff
@@ -0,0 +1,31 @@
+Name:     toggle-monitor
+Version:  1
+Release:  1
+Summary:  Script to turn monitor on and off
+License:  Public Domain
+
+Requires: xrandr
```

Comment:
Hello @kk-at-redhat  and thank you for changes. While reviewing, I noticed one more thing - the script uses the spd-say command which is part of speech-dispatcher-utils package. Please add requirement for this package.

Reply:
It's done.